code/datums/wires/seedstorage.dm

```diff
@@ -0,0 +1,60 @@
+/datum/wires/seedstorage
+	holder_type = /obj/machinery/seed_storage
+	wire_count = 4
+	random = 1
+
+var/const/SEED_WIRE_SMART = 1
```

Comment:
Do these need to be `var/const` as opposed to `#define`?
Defines use less memory (i.e., no memory as opposed to however much is required for a var) because the preprocessor parses them to their defined values at compile-time

And anyways global variables are generally bad.

Reply:
I've never touched wires before this, so I am not entirely certain on this. Will check.